src/frontend/src/lib/utils/tokens.utils.ts

```diff
@@ -124,33 +129,37 @@ export const sumTokensUiUsdBalance = (tokens: TokenUi[]): number =>
 	tokens.reduce((acc, token) => acc + (token.usdBalance ?? 0), 0);
 
 /**
- * Calculates total USD balance of mainnet tokens from the provided tokens list.
+ * Calculates total USD balance of mainnet tokens per network from the provided tokens list.
  *
  * @param $tokens - The list of tokens for filtering by network env and total USD balance calculation.
  * @param $balancesStore - The balances data for the tokens.
  * @param $exchanges - The exchange rates data for the tokens.
- * @returns The sum of mainnet tokens USD balance if balances and exchanges stores are available.
+ * @returns A NetworkId-number dictionary with total USD balance of mainnet tokens per network.
  *
  */
-export const sumMainnetTokensUsdBalance = ({
+export const calculateMainnetTokensUsdBalancesPerNetwork = ({
```

Comment:
`sumMainnetTokensUsdBalancesPerNetwork` would not have been correct too?

Reply:
Yes, renamed.